app/helpers/date_filter_helper.rb

```diff
@@ -27,4 +30,25 @@ def total_responses_header(total_count, from, to)
       response_total
     end
   end
+
+  def date_range_track_action(from_date=nil, to_date=nil)
+    if from_date && to_date
+      "filtering between two dates"
+    elsif from_date
+      "filtering everything since date"
+    elsif to_date
+      "filtering everying before date"
+    else
+      "no filtering"
+    end
+  end
+
+  def date_range_track_label(from_date, to_date)
+    from_date ||= Date.new(1970,1,1)
+    to_date ||= Date.current
+    date_range = distance_of_time_in_words(from_date, to_date)
+    date_ago = time_ago_in_words(to_date)
+    date_ago = 0 if to_date.at_middle_of_day() == Date.current.at_middle_of_day()
+    return "selected date range: #{date_range}, time between search and 'to' date provided: #{date_ago}"
```

Comment:
I think these need to be two discrete metrics being logged, perhaps as separate actions. I really don't know enough about what's possible for data extraction in GA, I think we need to properly research what's actually possible and discuss with Syed. I think we need to look at the existing GA account and see examples of this sort of data being extracted, and check what's possible.